src/dygraph-layout.js

```diff
@@ -247,12 +247,14 @@ DygraphLayout.prototype._evaluateLineCharts = function() {
     var axis = this.dygraph_.axisPropertiesForSeries(setName);
     // TODO (konigsberg): use optionsForAxis instead.
     var logscale = this.dygraph_.attributes_.getForSeries("logscale", setName);
+    var outOfXBounds = 0, outOfYBounds = 0;
 
     for (var j = 0; j < points.length; j++) {
       var point = points[j];
 
       // Range from 0-1 where 0 represents left and 1 represents right.
       point.x = DygraphLayout.calcXNormal_(point.xval, this._xAxis, isLogscaleForX);
+      outOfXBounds += ((point.x<0)||(point.x>1));
```

Comment:
nitpick `(point.x < 0) || (point.x > 1)`, I had left out the spaces in the mail to convey the idea and not go over normal eMail line length, and the outer parenthesēs are redundant here

Reply:
Done